src/helix-ui/elements/HXElement.js

```diff
@@ -92,7 +92,7 @@ export class HXElement extends HTMLElement {
 
         let evt = new CustomEvent(evtName, {
             bubbles: true,
-            details: details,
+            detail: details,
```

Comment:
We found a bug, and updated this one to hold the other attributes with the custom event